coinChange.java

```diff
@@ -0,0 +1,20 @@
+//Problem1 (https://leetcode.com/problems/coin-change/)
+T: O(MN), M=amount, N = number of coins
+S:O(M)
```

Comment:
Good Job, Please add the requested information as per the requested format along with an explanation
of your approach